changelog.md

```diff
@@ -1,6 +1,8 @@
 # MumieTask - Changelog
 
 All important changes to this plugin will be documented in this file.
+## [v2.2.0] 
+- Added Participant List
```

Comment:
We need to elaborate a bit on the details. Why was this added? Is this there a new feature a teacher could use?

For comparison, have a look at the moodle plugin's [changelog](https://github.com/integral-learning/moodle-mod_mumie/blob/master/changelog.md#added-3).


Furthermore, it should be listed under "added" as part of the upcoming release (right now it's v2.1.0)

Reply:
done